keras_hub/src/models/pali_gemma/pali_gemma_backbone.py

```diff
@@ -274,6 +274,13 @@ def __init__(
         # Keep the image_sequence_length as a backbone property for easy access.
         self.image_sequence_length = self.vit_encoder.image_sequence_length
 
+    def get_lora_target_names(self):
+        target_names = super().get_lora_target_names()
+
+        # Add these for `PaliGemmaVITAttention`.
+        target_names += ["query_proj", "value_proj"]
```

Comment:
Can we just align these names without breaking checkpoint compat? Fine to keep the overridable method, but seems like it would just save us headache (and improve overall ux) if we just kept the same naming convention here. There's no reason for the divergence right?